sdk/monitor/azure-monitor-ingestion/samples/async_samples/sample_upload_file_contents_async.py

```diff
@@ -0,0 +1,53 @@
+"""
+Upload a file containing a list of JSON objects to Azure Monitor.
+
+Usage: python sample_upload_file_contents_async.py
+"""
+import asyncio
+from datetime import datetime
+import json
+import os
+
+from azure.core.exceptions import HttpResponseError
+from azure.identity.aio import DefaultAzureCredential
+from azure.monitor.ingestion.aio import LogsIngestionClient
+
+
+async def upload_file_contents() -> None:
+    endpoint = os.environ['DATA_COLLECTION_ENDPOINT']
+    credential = DefaultAzureCredential()
+
+    client = LogsIngestionClient(endpoint=endpoint, credential=credential, logging_enable=True)
+
+    # Update this to point to a file containing a list of JSON objects.
+    FILE_PATH ="../../test-logs.json"
+
+    async with client:
+        # Option 1: Upload the file contents by passing in the file stream. With this option, no chunking is done, and the
+        # file contents are uploaded as is through one request. Subject to size service limits.
+        with open(FILE_PATH, "r") as f:
+            try:
+                await client.upload(
+                    rule_id=os.environ['LOGS_DCR_RULE_ID'],
+                    stream_name=os.environ['LOGS_DCR_STREAM_NAME'],
+                    logs=f)
+            except HttpResponseError as e:
+                print(f"Upload failed: {e}")
```

Comment:
When running the sample it might be hard to tell which one failed. Maybe update to "File stream upload failed:" etc

Reply:
Good call. updated!